docs/releases/1.11.txt

```diff
@@ -314,6 +314,9 @@ Management Commands
 * The new :option:`diffsettings --default` option allows specifying a settings
   module other than Django's default settings to compare against.
 
+* Improved the way the :djadmin:`shell` command reads python code from standard
```

Comment:
I'm not sure a release note is needed -- it's not really clear to me what it means. Improved? How so? If some doc update were made in `ref/django-admin.txt` to describe the new ability, then you could link to it from here.